server/routes/chat_routes.ts

```diff
@@ -329,7 +326,7 @@ export function registerChatRoutes(router: IRouter, routeOptions: RoutesOptions)
       request,
       response
     ): Promise<IOpenSearchDashboardsResponse<HttpResponsePayload | ResponseError>> => {
-      const chatService = createChatService();
+      const chatService = createChatService(context);
 
       try {
         chatService.abortAgentExecution(request.body.sessionId);
```

Comment:
It seems that we can refactor the `abortAgentExecution` to a static method. So we don't need to create ChatService instance anymore.

Reply:
The method `abortAgentExecution ` here implements the method in the interface `ChatService`, it may not be a static method?

Reply:
It seems the abortAgentExecution won't abort the agent for now. It's ok to me to keep current implementation. Maybe the abort method will depends on context or agentName in the future.